From 0d821053aa4a3f265a5768c8bf51eaab9a81de03 Mon Sep 17 00:00:00 2001
From: Joey <joey@lineageos.org>
Date: Sat, 19 May 2018 22:38:06 +0200
Subject: [PATCH 5/5] Recorder: update assets to use rounded style

The app uses has a string usage of rounded elements,
let's make all the icons follow this style

Change-Id: I8176ea5c1b75e85f7ff6a51cf923fd13745b0c3d
Signed-off-by: Joey <joey@lineageos.org>
---
 app/src/main/res/drawable/ic_action_screen_record.xml | 2 +-
 app/src/main/res/drawable/ic_action_sound_record.xml  | 5 ++++-
 app/src/main/res/drawable/ic_delete.xml               | 2 +-
 app/src/main/res/drawable/ic_drag.xml                 | 2 +-
 app/src/main/res/drawable/ic_last_screen.xml          | 2 +-
 app/src/main/res/drawable/ic_last_sound.xml           | 2 +-
 app/src/main/res/drawable/ic_play.xml                 | 2 +-
 app/src/main/res/drawable/ic_stop.xml                 | 2 +-
 app/src/main/res/drawable/ic_stop_screen.xml          | 2 +-
 app/src/main/res/drawable/ic_stop_sound.xml           | 2 +-
 10 files changed, 13 insertions(+), 10 deletions(-)

diff --git a/app/src/main/res/drawable/ic_action_screen_record.xml b/app/src/main/res/drawable/ic_action_screen_record.xml
index c9036a2..6f77bc8 100644
--- a/app/src/main/res/drawable/ic_action_screen_record.xml
+++ b/app/src/main/res/drawable/ic_action_screen_record.xml
@@ -20,5 +20,5 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/screen"
-        android:pathData="M17,10.5V7c0,-0.55 -0.45,-1 -1,-1H4c-0.55,0 -1,0.45 -1,1v10c0,0.55 0.45,1 1,1h12c0.55,0 1,-0.45 1,-1v-3.5l4,4v-11l-4,4z" />
+        android:pathData="M17,10.5V7c0,-0.55 -0.45,-1 -1,-1H4C3.45,6 3,6.45 3,7v10c0,0.55 0.45,1 1,1h12c0.55,0 1,-0.45 1,-1v-3.5l2.29,2.29c0.63,0.63 1.71,0.18 1.71,-0.71V8.91c0,-0.89 -1.08,-1.34 -1.71,-0.71L17,10.5z"/>
 </vector>
diff --git a/app/src/main/res/drawable/ic_action_sound_record.xml b/app/src/main/res/drawable/ic_action_sound_record.xml
index bbe35cd..2443c9f 100644
--- a/app/src/main/res/drawable/ic_action_sound_record.xml
+++ b/app/src/main/res/drawable/ic_action_sound_record.xml
@@ -20,5 +20,8 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/sound"
-        android:pathData="M12,14c1.66,0 2.99,-1.34 2.99,-3L15,5c0,-1.66 -1.34,-3 -3,-3S9,3.34 9,5v6c0,1.66 1.34,3 3,3zM17.3,11c0,3 -2.54,5.1 -5.3,5.1S6.7,14 6.7,11L5,11c0,3.41 2.72,6.23 6,6.72L11,21h2v-3.28c3.28,-0.48 6,-3.3 6,-6.72h-1.7z" />
+        android:pathData="M12,14c1.66,0 3,-1.34 3,-3V5c0,-1.66 -1.34,-3 -3,-3S9,3.34 9,5v6C9,12.66 10.34,14 12,14z"/>
+    <path
+        android:fillColor="@color/sound"
+        android:pathData="M17.91,11c-0.49,0 -0.9,0.36 -0.98,0.85C16.52,14.2 14.47,16 12,16s-4.52,-1.8 -4.93,-4.15C6.99,11.36 6.58,11 6.09,11h0c-0.61,0 -1.09,0.54 -1,1.14c0.49,3 2.89,5.35 5.91,5.78V20c0,0.55 0.45,1 1,1h0c0.55,0 1,-0.45 1,-1v-2.08c3.02,-0.43 5.42,-2.78 5.91,-5.78C19.01,11.54 18.52,11 17.91,11L17.91,11z"/>
 </vector>
diff --git a/app/src/main/res/drawable/ic_delete.xml b/app/src/main/res/drawable/ic_delete.xml
index f4ce3d4..6a2a66d 100644
--- a/app/src/main/res/drawable/ic_delete.xml
+++ b/app/src/main/res/drawable/ic_delete.xml
@@ -20,5 +20,5 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/black"
-        android:pathData="M6,19c0,1.1 0.9,2 2,2h8c1.1,0 2,-0.9 2,-2V7H6v12zM19,4h-3.5l-1,-1h-5l-1,1H5v2h14V4z" />
+        android:pathData="M6,19c0,1.1 0.9,2 2,2h8c1.1,0 2,-0.9 2,-2V9c0,-1.1 -0.9,-2 -2,-2H8C6.9,7 6,7.9 6,9V19zM18,4h-2.5l-0.71,-0.71C14.61,3.11 14.35,3 14.09,3H9.91C9.65,3 9.39,3.11 9.21,3.29L8.5,4H6C5.45,4 5,4.45 5,5v0c0,0.55 0.45,1 1,1h12c0.55,0 1,-0.45 1,-1v0C19,4.45 18.55,4 18,4z" />
 </vector>
diff --git a/app/src/main/res/drawable/ic_drag.xml b/app/src/main/res/drawable/ic_drag.xml
index 9ce9c0f..0423984 100644
--- a/app/src/main/res/drawable/ic_drag.xml
+++ b/app/src/main/res/drawable/ic_drag.xml
@@ -20,5 +20,5 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/white"
-        android:pathData="M13,11H18L16.5,9.5L17.92,8.08L21.84,12L17.92,15.92L16.5,14.5L18,13H13V18L14.5,16.5L15.92,17.92L12,21.84L8.08,17.92L9.5,16.5L11,18V13H6L7.5,14.5L6.08,15.92L2.16,12L6.08,8.08L7.5,9.5L6,11H11V6L9.5,7.5L8.08,6.08L12,2.16L15.92,6.08L14.5,7.5L13,6V11Z" />
+        android:pathData="M10.5,9h3C13.78,9 14,8.78 14,8.5V6h1.79c0.45,0 0.67,-0.54 0.35,-0.85l-3.79,-3.79c-0.2,-0.2 -0.51,-0.2 -0.71,0L7.85,5.15C7.54,5.46 7.76,6 8.21,6H10v2.5C10,8.78 10.22,9 10.5,9zM8.5,10H6V8.21c0,-0.45 -0.54,-0.67 -0.85,-0.35l-3.79,3.79c-0.2,0.2 -0.2,0.51 0,0.71l3.79,3.79C5.46,16.46 6,16.24 6,15.79V14h2.5C8.78,14 9,13.78 9,13.5v-3C9,10.22 8.78,10 8.5,10zM22.65,11.65l-3.79,-3.79C18.54,7.54 18,7.76 18,8.21V10h-2.5c-0.28,0 -0.5,0.22 -0.5,0.5v3c0,0.28 0.22,0.5 0.5,0.5H18v1.79c0,0.45 0.54,0.67 0.85,0.35l3.79,-3.79C22.84,12.16 22.84,11.84 22.65,11.65zM13.5,15h-3c-0.28,0 -0.5,0.22 -0.5,0.5V18H8.21c-0.45,0 -0.67,0.54 -0.35,0.85l3.79,3.79c0.2,0.2 0.51,0.2 0.71,0l3.79,-3.79c0.31,-0.31 0.09,-0.85 -0.35,-0.85H14v-2.5C14,15.22 13.78,15 13.5,15z" />
 </vector>
diff --git a/app/src/main/res/drawable/ic_last_screen.xml b/app/src/main/res/drawable/ic_last_screen.xml
index 5f9f2f8..8f917ec 100644
--- a/app/src/main/res/drawable/ic_last_screen.xml
+++ b/app/src/main/res/drawable/ic_last_screen.xml
@@ -20,5 +20,5 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/icon"
-        android:pathData="M18,4l2,4h-3l-2,-4h-2l2,4h-3l-2,-4H8l2,4H7L5,4H4c-1.1,0 -1.99,0.9 -1.99,2L2,18c0,1.1 0.9,2 2,2h16c1.1,0 2,-0.9 2,-2V4h-4z" />
+        android:pathData="M18,4l1.82,3.64C19.9,7.8 19.78,8 19.6,8h-1.98c-0.38,0 -0.73,-0.21 -0.89,-0.55L15,4h-2l1.82,3.64C14.9,7.8 14.78,8 14.6,8h-1.98c-0.38,0 -0.73,-0.21 -0.89,-0.55L10,4H8l1.82,3.64C9.9,7.8 9.78,8 9.6,8H7.62C7.24,8 6.89,7.79 6.72,7.45L5,4H4C2.9,4 2,4.9 2,6v12c0,1.1 0.9,2 2,2h16c1.1,0 2,-0.9 2,-2V5c0,-0.55 -0.45,-1 -1,-1H18z" />
 </vector>
diff --git a/app/src/main/res/drawable/ic_last_sound.xml b/app/src/main/res/drawable/ic_last_sound.xml
index 448e1be..9be9f42 100644
--- a/app/src/main/res/drawable/ic_last_sound.xml
+++ b/app/src/main/res/drawable/ic_last_sound.xml
@@ -20,5 +20,5 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/icon"
-        android:pathData="M20,4L4,4c-1.1,0 -2,0.9 -2,2v12c0,1.1 0.9,2 2,2h16c1.1,0 2,-0.9 2,-2L22,6c0,-1.1 -0.9,-2 -2,-2zM7.76,16.24l-1.41,1.41C4.78,16.1 4,14.05 4,12c0,-2.05 0.78,-4.1 2.34,-5.66l1.41,1.41C6.59,8.93 6,10.46 6,12s0.59,3.07 1.76,4.24zM12,16c-2.21,0 -4,-1.79 -4,-4s1.79,-4 4,-4 4,1.79 4,4 -1.79,4 -4,4zM17.66,17.66l-1.41,-1.41C17.41,15.07 18,13.54 18,12s-0.59,-3.07 -1.76,-4.24l1.41,-1.41C19.22,7.9 20,9.95 20,12c0,2.05 -0.78,4.1 -2.34,5.66zM12,10c-1.1,0 -2,0.9 -2,2s0.9,2 2,2 2,-0.9 2,-2 -0.9,-2 -2,-2z" />
+        android:pathData="M20,4H4C2.9,4 2,4.9 2,6v12c0,1.1 0.9,2 2,2h16c1.1,0 2,-0.9 2,-2V6C22,4.9 21.1,4 20,4zM7.11,16.89L7.11,16.89c-0.43,0.43 -1.14,0.39 -1.51,-0.09C4.53,15.39 4,13.69 4,12c0,-1.69 0.53,-3.38 1.59,-4.8C5.96,6.72 6.67,6.67 7.1,7.1l0,0C7.45,7.45 7.49,8 7.2,8.39C6.4,9.46 6,10.73 6,12c0,1.27 0.4,2.53 1.2,3.6C7.5,15.99 7.46,16.54 7.11,16.89zM12,16c-2.21,0 -4,-1.79 -4,-4s1.79,-4 4,-4s4,1.79 4,4S14.21,16 12,16zM16.9,16.9L16.9,16.9c-0.35,-0.35 -0.39,-0.9 -0.09,-1.29C17.6,14.54 18,13.27 18,12c0,-1.27 -0.4,-2.53 -1.2,-3.6c-0.3,-0.39 -0.26,-0.95 0.09,-1.3l0,0c0.43,-0.43 1.14,-0.39 1.51,0.09c1.07,1.41 1.6,3.1 1.6,4.8c0,1.69 -0.53,3.38 -1.59,4.8C18.04,17.28 17.33,17.33 16.9,16.9zM12,10c-1.1,0 -2,0.9 -2,2s0.9,2 2,2s2,-0.9 2,-2S13.1,10 12,10z"/>
 </vector>
diff --git a/app/src/main/res/drawable/ic_play.xml b/app/src/main/res/drawable/ic_play.xml
index c1b4003..d05c0f1 100644
--- a/app/src/main/res/drawable/ic_play.xml
+++ b/app/src/main/res/drawable/ic_play.xml
@@ -20,5 +20,5 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/black"
-        android:pathData="M8,5v14l11,-7z" />
+        android:pathData="M8,6.82v10.36c0,0.79 0.87,1.27 1.54,0.84l8.14,-5.18c0.62,-0.39 0.62,-1.29 0,-1.69L9.54,5.98C8.87,5.55 8,6.03 8,6.82z" />
 </vector>
diff --git a/app/src/main/res/drawable/ic_stop.xml b/app/src/main/res/drawable/ic_stop.xml
index 28084c7..186d91e 100644
--- a/app/src/main/res/drawable/ic_stop.xml
+++ b/app/src/main/res/drawable/ic_stop.xml
@@ -20,5 +20,5 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/black"
-        android:pathData="M6,6h12v12H6z"/>
+        android:pathData="M8,6h8c1.1,0 2,0.9 2,2v8c0,1.1 -0.9,2 -2,2H8c-1.1,0 -2,-0.9 -2,-2V8C6,6.9 6.9,6 8,6z"/>
 </vector>
diff --git a/app/src/main/res/drawable/ic_stop_screen.xml b/app/src/main/res/drawable/ic_stop_screen.xml
index 617dfa6..9106c42 100644
--- a/app/src/main/res/drawable/ic_stop_screen.xml
+++ b/app/src/main/res/drawable/ic_stop_screen.xml
@@ -20,5 +20,5 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/screen"
-        android:pathData="M6,6h12v12H6z" />
+        android:pathData="M8,6h8c1.1,0 2,0.9 2,2v8c0,1.1 -0.9,2 -2,2H8c-1.1,0 -2,-0.9 -2,-2V8C6,6.9 6.9,6 8,6z"/>
 </vector>
diff --git a/app/src/main/res/drawable/ic_stop_sound.xml b/app/src/main/res/drawable/ic_stop_sound.xml
index f71e79e..e46a579 100644
--- a/app/src/main/res/drawable/ic_stop_sound.xml
+++ b/app/src/main/res/drawable/ic_stop_sound.xml
@@ -20,5 +20,5 @@
     android:viewportWidth="24">
     <path
         android:fillColor="@color/sound"
-        android:pathData="M6,6h12v12H6z" />
+        android:pathData="M8,6h8c1.1,0 2,0.9 2,2v8c0,1.1 -0.9,2 -2,2H8c-1.1,0 -2,-0.9 -2,-2V8C6,6.9 6.9,6 8,6z"/>
 </vector>
-- 
2.17.0

